overlay.d/25rhcos-azure-udev-rules/usr/lib/udev/rules.d/50-azure-ptp.rules

```diff
@@ -0,0 +1,5 @@
+# Manual backport of:
+# https://github.com/systemd/systemd/commit/32e868f058da8b90add00b2958c516241c532b70
+# Can drop once we get to RHEL 8.6:
```

Comment:
> # Can drop once we get to RHEL 8.6:

Is there anything we can put in place now to force us to do that?

Reply:
seems like a ext test that said `if RHEL 8.6 && -f  50-azure-ptp.rules; then fail`

Reply:
I'm thinking we should extend kola to handle these kinds of checks so they're non-fatal but still loud. Some ideas:
- have kola check test output for `KOLAWARNING: ...` and reprint the warning on the console.
- add a `warn-on-rc: N` field to kola test JSON where if a test exits with the given code, kola prints `WARN: $testname` instead of `FAIL: $testname`.

We can then have kola e.g. touch a file or print a report of warnings and the Slack notifier in the pipelines can check for this file and tweak the notification.

Reply:
carried forward discussion in https://github.com/coreos/coreos-assembler/issues/2620